src/content/docs/en/editor-setup.mdx

```diff
@@ -28,7 +28,7 @@ import ReadMore from '~/components/ReadMore.astro';
 
 Initial support for Astro landed in WebStorm 2023.1. You can install the official plugin through [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/20959-astro) or by searching for "Astro" in the IDE's Plugins tab. This plugin includes features like syntax highlighting, code completion, and formatting, and plans to add even more advanced features in the future. It is also available to all other [JetBrains IDEs with JavaScript support](https://www.jetbrains.com/products/#lang=js&type=ide).
 
-JetBrains' upcoming [Fleet IDE](https://www.jetbrains.com/fleet/) includes language server support and will be able to use the official Astro tooling.
+JetBrains' [Fleet IDE](https://www.jetbrains.com/fleet/) includes language server support and uses the official Astro tooling.
```

Comment:
This is not really true, Fleet still requires a plugin (that doesn't exist). It's just intended to be compatible with the underlying technology we use and so developing a plugin is *possible*

Though, I'm not even sure if it has its LSP support released yet.

Reply:
OK this is causing me way too much grief. FLEET IS DEAD TO ME.

Reply:
I'm all for getting out the pitchforks 😄